environment

```diff
@@ -1,9 +1,42 @@
-#!/usr/bin/env bash
+# Resolve the location of this file and set APP_HOME to the root
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ] ; do SOURCE="$(readlink "$SOURCE")"; done
+export APP_HOME="$(cd -P "$(dirname "$SOURCE")" && pwd)"
+
 set -a
-PROJECT_ROOT=`pwd`
-SERVICE_NAME='query'
-DCP_DNS_DOMAIN='data.humancellatlas.org'
-DEPLOYMENT_STAGE="${DEPLOYMENT_STAGE:-dev}"
-AWS_DEFAULT_REGION='us-east-1'
-API_HOST=${SERVICE_NAME}.${DEPLOYMENT_STAGE}.${DCP_DNS_DOMAIN}
+APP_NAME=dcpquery
+STAGE="${DEPLOYMENT_STAGE:-dev}"
+if [[ $STAGE == "prod" ]]; then
+   API_DOMAIN_NAME=query.data.humancellatlas.org
+   API_DNS_ZONE=data.humancellatlas.org.
+else
+   API_DOMAIN_NAME=query.${STAGE}.data.humancellatlas.org
+   API_DNS_ZONE=${STAGE}.data.humancellatlas.org.
+fi
+BUNDLE_EVENTS_QUEUE_NAME=${APP_NAME}-${STAGE}-bundle-events
+ASYNC_QUERIES_QUEUE_NAME=${APP_NAME}-${STAGE}-async-queries
+WEBHOOK_SECRET_NAME=${APP_NAME}/${STAGE}/webhook-auth-config
+
+# Set DEBUG=0 to disable debugging and set the app log level to ERROR.
+# Set DEBUG=1 to change the log level to INFO and cause stack traces to appear in error responses.
+# Set DEBUG=2 to change the log level to DEBUG and cause stack traces to appear in error responses.
+DEBUG=1
+
+TFSTATE_FILE=.terraform/remote.tfstate
+TF_CLI_ARGS_output="--state ${TFSTATE_FILE}"
+TF_CLI_ARGS_init="--backend-config ${APP_HOME}/.terraform/aws_config"
+
+if [[ ! -v AWS_DEFAULT_REGION ]]; then
```

Comment:
```
> source environment 
-bash: environment: line 29: conditional binary operator expected
-bash: environment: line 29: syntax error near `AWS_DEFAULT_REGION'
-bash: environment: line 29: `if [[ ! -v AWS_DEFAULT_REGION ]]; then'
```

Reply:
It looks like my bash is too new :) Fixed, sorry about that.